src/camera.rs

```diff
@@ -106,7 +106,10 @@ impl Camera for Camera2D {
     }
 
     fn viewport(&self) -> Option<(i32, i32, i32, i32)> {
-        self.viewport
+        match self.viewport {
```

Comment:
```
let dpi_scale = ..;
self.viewport.map(|(x, y, w, h)| (x * dpi_scale, ..))
```

will look a little better :)

Reply:
also I am not really sure if `camera.viewport` and `camera.viewport()` returning different values is the right way to fix it

Reply:
> ```
> let dpi_scale = ..;
> self.viewport.map(|(x, y, w, h)| (x * dpi_scale, ..))
> ```
> 
> will look a little better :)

I thought there would have been a better way but I wasn't quite sure what it was, I will add this now

Reply:
> also I am not really sure if `camera.viewport` and `camera.viewport()` returning different values is the right way to fix it

Possibly not, probably best to change where it is instantiated I guess

Reply:
> > ```
> > let dpi_scale = ..;
> > self.viewport.map(|(x, y, w, h)| (x * dpi_scale, ..))
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > will look a little better :)
> 
> I thought there would have been a better way but I wasn't quite sure what it was, I will add this now

Had to add them all individually

Reply:
> > also I am not really sure if `camera.viewport` and `camera.viewport()` returning different values is the right way to fix it
> 
> Possibly not, probably best to change where it is instantiated I guess

It is instantiated when the camera is made, so unless I'm missing something it can't be modified when made